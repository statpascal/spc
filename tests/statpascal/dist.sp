program dist;

function l2_dist (a, b: realvector): real;
    begin
        l2_dist := sqrt (sum (sqr (a - b)))
    end;

begin
    writeln (l2_dist (combine (1, 1, 1), combine (2, 2, 2)), ' ', sqrt (3.0))
end.
