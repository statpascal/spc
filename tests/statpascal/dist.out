1.73205 1.73205
